pkg/apis/dynatrace/v1alpha1/defaults.go

```diff
@@ -1,5 +1,12 @@
 package v1alpha1
 
+import (
+	"fmt"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
 func SetDefaults_OneAgentSpec(obj *OneAgentSpec) {
```

Comment:
The `obj` should be `cr` or something similar.

Reply:
Right, but the stub for this function got generated by `defaulter-gen`. The name `obj` is used throughout the whole Kubernetes source:
https://github.com/search?q=org%3Akubernetes+%22func+SetDefaults_%22&type=Code